docs/databases/swift-kuery.html

```diff
@@ -21,37 +21,44 @@
+      <p class="block-text"><a href="https://github.com/IBM-Swift/Swift-Kuery" target="_blank">Swift-Kuery</a> is a pluggable SQL database driver/SDK abstraction layer.
+           Its main idea is to unify the APIs to the various relational databases,
+            providing a Swifty yet SQL-like API.
+            This guide will demonstrate how to connect to a SQL Database using one of the Swift-Kuery plugins and how to use this connection to send SQL queries to your database.</p>
       <h2 class="heading-2">Prerequisites</h2>
       <ul class="plain-list">
-        <li><span class="blue-text">Kitura Server:</span> Learn how to create one in our Getting Started guide</li>
-        <li><span class="blue-text">Logging:</span> Learn how to add logging to your project</li>
-        <li><a onclick="localStorage.setItem('package', 'Swift-Kuery'); setActiveSidebarElementForParent('swift-kuery')" href="../getting-started/update-package.html#">Update Dependencies:</a> Swift Kuery needs adding to the Package.swift file.</li>
+          <li><a onclick="setActiveSidebarElementForParent('create-server')" href="../getting-started/create-server.html#">Kitura Server:</a> Learn how to create one in our Getting Started guide.</li>
+          <li><a onclick="setActiveSidebarElementForParent('style-guide')" href="../getting-started/style-guide.html#">Kitura Style Guide:</a> Follow our guide to see how a production ready server should be structured.</li>
+          <li><a onclick="setActiveSidebarElementForParent('style-guide')" href="../getting-started/style-guide.html#">Book Model:</a> You have defined a `Book` struct as described in Step 5 of the style guide.</li>
+          <li><a onclick="setActiveSidebarElementForParent('logging')" href="../getting-started/logging.html#">Logging:</a> Learn how to add logging to your project.</li>
+          <li><a onclick="localStorage.setItem('package', 'Swift-Kuery'); setActiveSidebarElementForParent('swift-kuery')" href="../getting-started/update-package.html#">Update Dependencies:</a> Your Swift Kuery plugin needs to be added to the Package.swift file.</li>
       </ul>
       <div class="underline"></div>
-      <h2 class="heading-2"><span class="blue-text">Step 1:</span> Create an `KueryRoutes.swift` file</h2>
+      <h2 class="heading-2"><span class="blue-text">Step 1:</span> Create the Kuery routes</h2>
       <p>Open your `Sources` > `Application` > `Application.swift`</p>
       <p> Inside the `postInit()` function add: </p>
       <pre><code class="language-swift">initializeKueryRoutes(app: self)</code></pre>
       <p>Create a new file called `KueryRoutes.swift` in `Sources` > `Application` > `Routes`</p>
       <p>Inside this file, add the framework for our routes code:</p>
```

Comment:
I think we should have a few more words here, like "we're going to create xx to handle selecting and inserting xx" whatever it is we're going to demonstrate.